homeassistant/components/template/coordinator.py

```diff
@@ -0,0 +1,81 @@
+"""The template update coordinator."""
```

Comment:
This doesn't explain well what the coordinator does. @gjohansson-ST can you maybe suggest something?